content/articles/setting-up-hotspot-on-kali-linux/index.md

```diff
@@ -0,0 +1,134 @@
+﻿###  Setting Up Hotspot On Kali Linux
+
+Kali Linux is considered the best operating system in cyber security implementation by both white hat and black hat hackers.
+Each distro of Linux has an intended purpose for implementation.
+
+### Introduction
+
+Kali Linux distro comes with no mobile hotspot ability due to security issues (makes it less vulnerable to attackers), creating a mobile hotspot can only be done by customizing the network configuration.
+In this article, we will look into the processes that we will use to create a mobile  hotspot on Kali Linux and other Linux distros like Ubuntu. Throughout this tutorial, we are going to be using Kali Linux to create our hotspot. A person using any other distribution can follow through.
+
```

Comment:
@lizpart Your tutorial seems to be extremely light on substance in this highlighted areas. It's too summarized. Please add some more content in these sections. Explain points in detail.

You also need to add a "Table of Content" section.